### ISSUE TITLE
Can't solve with moving head examples

### ISSUE THREAD

**Issue by 2000dB** [elliotwoods/ofxCeres#5]
I'm trying to use the moving head calibration examples as a starting point but I can't get a solve result, so I'm wondering if I'm doing something wrong.

Target x64 debug, the examples compile nicely.
If I click 'add test data' and solve, I get the following report. Is this the expected result? I assume it's not... 

Tried with my own target points and pan/tilt values and I have some issue there as well but I would like to sanity check with the test data first.

Thanks!

`Solver Summary (v 1.12.0-eigen-(3.3.90)-no_lapack-eigensparse-openmp)

                                     Original                  Reduced
Parameter blocks                            3                        3
Parameters                                 13                       13
Residual blocks                            44                       44
Residual                                   44                       44

Minimizer                        TRUST_REGION

Dense linear algebra library            EIGEN
Trust region strategy     LEVENBERG_MARQUARDT

                                        Given                     Used
Linear solver                        DENSE_QR                 DENSE_QR
Threads                                     1                        1
Linear solver threads                       1                        1
Linear solver ordering              AUTOMATIC                        3

Cost:
Initial                         -1.000000e+00

Minimizer iterations                        0
Successful steps                            0
Unsuccessful steps                          0

Time (in seconds):
Preprocessor                           0.0001

  Residual evaluation                  0.0000
  Jacobian evaluation                  0.0013
  Linear solver                        0.0000
Minimizer                              0.0015

Postprocessor                          0.0000
Total                                  0.0017

Termination:                          FAILURE (Residual and Jacobian evaluation failed.)`

**Comment by elliotwoods** [comment]
hmm
I'm not sure what the issue is from seeing this error
there might be a NaN in there
e.g. if one of your control points is xyz={0,0,0} and the initial parameters put the moving head at the same position, then you'll get a NaN

Maybe that's the first thing to check (that the initial position != one of your target positions)

**Comment by 2000dB** [comment]
Hi Elliot,

Thanks for the quick response.

I doubled checked that no points were at 0,0,0. It's not the case. I'm using the baked in test data from your example.
I've reduced the number of points to the ones labeled A-G now and I get a solution but all the translation and rotation transformations are all 0s except for tilt offset which is quite high. See attached screenshot taken after hitting the solve button.

Many thanks

![Capture](https://user-images.githubusercontent.com/202499/120497772-f9f94e80-c3be-11eb-9095-08fcb99f389d.PNG)

**Comment by elliotwoods** [comment]
If I get time tommrow I'll have a quick look at the example
I changed w lot of things in ofxCeres recently, so it might be worth trying to check out an earlier version (eg the last commit for that example )

**Comment by 2000dB** [comment]
Indeed, it seems to solve fine in a previous checkout. I'll have a closer look on Friday.
